### PR TITLE
Fix ipad pdf issue

### PR DESCRIPTION
**Pull request by asterns**
This fixes the ipad issue by constructing a short-lived url containing a jwt that displays the pdf, and then does a window.open.  On most browsers, this will flash open a new tab and then close it immediately and download the pdf after that.  Not the ideal user experience, but probably fine.  For chrome on ipad, it opens a new tab and just keeps it there where the user can print.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ variables:
 	REACT_APP_RECAPTCHA_SITE_KEY=6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
 	REACT_APP_RECAPTCHA_SECRET_KEY=6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe
 	REACT_APP_SLACK_WEBHOOK_URL=<LEAVEBLANK>
+	REACT_APP_JWT_SECRET=eXB62QzaWp54hrjHtyfYJVgwr5NJNZ5dCRf43wEj
```
Comment (sjforman) [inline README.md]: This is arbitrary, I assume, so, this secret can be used for dev, and I'll generate a different string for prod use?

**Comment by sjforman** [comment]
Note on my comment above: this was on iOS 11. Now upgrading to iOS 12 to see if I have better luck there.

**Comment by asterns** [comment]
I had to fudge the security a bit for testing on the iPad because I wasn't
able to log in because the redirect URI on my network was not registered at
auth0. I'm driving right now so can't really look very closely at it will
look later tonight

File: README.md (hunk 1 above)
Comment (asterns) [reply README.md]: Correct - this is an arbitrary secret and should be set to something different in production

**Comment by sjforman** [comment]
@asterns Ah, gotcha, ok. And yes, the fact that I can't seem to find a way to test this isn't because of any change you make. Just haven't ever tried to access local env from ipad before. 

My process was similar / analogous: 

1) start server
2) `ngrok http 3000`
3) navigate on iOS safari to the proxied url provided by ngrok
4) sign in
5) when redirected after sign-in to localhost:3000//`stuff`, replace that `localhost:3000` with the ngrok url instead. 

This worked, to sign in (I think it would probably work for you too). 

But then, the API call getDistricts failed because of what sounds to me like a CORS-related problem. So then I tried fixing this typo — https://github.com/sjforman/votefwd/commit/be1186ef837e78d25370253586d96a763ff79595 — still no luck. 

I'm absolutely drowning in new users at the moment, so, @jeffkole if at all possible, it would be *enormously* helpful if you could take a look at this and see if you can conclude to your satisfaction that the fix worked for you as well, and then if so, approve. I'll figure out how to run locally on iOS at some point when I can catch my breath. thank you both

**Comment by jeffkole** [comment]
I don't have an iPad at all (well, I have a first generation one running outdated Safari, so it probably won't be helpful here).  Regardless, I tried testing on my iPhone, but the UI doesn't seem to work, even though it looks just fine in the Chrome device simulator.  Maybe it is the CORS issues you are seeing.  I'll try playing around some more, but I'm a bit miffed.  Short story, I doubt I'll be able to validate the changes tonight.

**Comment by sjforman** [comment]
Hm, Ok. Because I'm pretty baffled as to how to test this myself, but I _can_ see that downloading still works as normal on laptop — so hopefully this won't make things worse! — I'm going to merge and deploy and see if it does the trick, with fingers crossed. Can always revert if necessary. I saw the change you just pushed, @asterns, which looks good. Anything else you think needs doing before we try this in the wild?

**Comment by asterns** [comment]
@jeffkole - can you confirm you are changing `REACT_APP_URL` and `REACT_APP_API_URL` to whatever url you're pointing to on the iphone?  It seems odd there would be a CORS issue if the api being called was to the same domain as the original page the iphone loaded.

**Comment by jeffkole** [comment]
May have spoken too soon.

**Comment by jeffkole** [comment]
Got the CORS stuff settled so I can load the page.  On desktop Chrome, the pop-up is blocked and nothing happens.  That doesn't seem like a good solution.

On the iPhone, I get the same behavior as on `master`.  Click the "Download letter" link and nothing happens.

**Comment by jeffkole** [comment]
FWIW, just setting the `Content-Type` header didn't fix the problem with no other changes.

**Comment by asterns** [comment]
Darn - pop-up blockers could be a problem. Let me try something...

**Comment by asterns** [comment]
Ok, I made a slight change.  The exact mechanics of it are kind of poor form in React, but I couldn't quite figure out a way around it.  I don't think it will break anything, just kind of frowned upon to modify the dom at all in react.  If anyone knows a better way to accomplish this please let me know. It no longer opens a new window, and I'm hopeful it would work in iPad/iPhone. @jeffkole can you check your iPhone please?

**Comment by asterns** [comment]
@jeffkole @sjforman - I was able to borrow someone's iphone to test the most recent change, and it works.  It brings the pdf up rather than downloading it, but I think that's fine.

**Comment by sjforman** [comment]
Awesome. THANK YOU, @jeffkole and @asterns. I'll merge and deploy now with fingers crossed.

**Pull request merged**